sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -7899,7 +7898,34 @@
               "tableView": false,
               "components": [
                 {
-                  "label": "Will you be paying room and board (rent) to your parent(s)/step-parent/sponsor/legal guardian?",
+                  "label": "Are you living independently in a self-contained suite?",
+                  "optionsLabelPosition": "right",
+                  "tooltip": "The suite has a separate entrance, kitchen, bathroom and living area/bedroom; and\r\n\r\nYou are paying fair market rent and responsible for your share of the utilities (hydro, telephone, cable, etc. if not included in rent).",
```

Comment:
A single `\n` instead of `\r\n\r\n` would be enough, and following other tooltips.

Reply:
Thanks, and updated! Was my error in the form editor.